projects/ngx-charts-on-fhir/src/lib/fhir-chart/medication-scale.spec.ts

```diff
@@ -44,9 +44,11 @@ describe('MedicationScale', () => {
         scales: {
           category: {
             type: 'category',
+            axis:'r'
           },
           medication: {
             type: 'medication',
+            axis:'r'
           },
         },
```

Comment:
Why did you use 'r' instead of 'y' here? The unit test doesn't care, but it's a bit misleading because we don't support radial plots and I doubt the medication scale will display correctly if you try to use it on a radial axis.

Reply:
okay @semicolin I will fix this.